packages/block-editor/src/hooks/color-panel.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import PanelColorSettings from '../components/panel-color-settings';
+import ContrastChecker from '../components/contrast-checker';
+import InspectorControls from '../components/inspector-controls';
+import { getBlockDOMNode } from '../utils/dom';
+
+export default function ColorPanel( { colorSettings, clientId } ) {
```

Comment:
I this this should live in `block-editor/src/components` instead.

Reply:
We have so many color related components there that need some cleanup, maybe we should avoid a new reusable component there for now. Potentially the `hooks/color` could be a folder?